src/hooks/useActiveNamespace.ts

```diff
@@ -9,7 +9,7 @@ export const useActiveNamespace = () => {
 
   useEffect(() => {
     const fetchProjects = async () => {
-      const projects = await k8sListResource({
+      const { items: projects }: any = await k8sListResource({
```

Comment:
There is `k8sListResourceItems`: https://github.com/openshift/dynamic-plugin-sdk/blob/main/packages/lib-utils/src/k8s/k8s-resource.ts#L152

No need to destruct it.

Reply:
Oooh! Okay, will do that!